docutils/plan.sh

```diff
@@ -25,4 +29,20 @@ do_build() {
 
 do_install() {
   python install.py
+
+  # write our wrapper script
+  bash_path=$(hab pkg path core/bash)
```

Comment:
You'll want to use `$(pkg_path_for core/bash)` to be certain you get the version of bash that is in your pkg_deps.

Reply:
Done!

I'm still not sure I understand the difference.  I get why you were saying not to use `$(hab pkg path core/bash)` directly in the actual wrapper script.  But here they should do the same thing, no?

Reply:
`pkg_path_for` uses `pkg_deps` to determine what version of the named package to use. This will allow us to guard against that dep being accidentally removed and will always give us the version that will also be installed with this package.

Reply:
ah! gotcha!  TIL, thanks!